docs/feature-compatibility.txt

```diff
@@ -63,6 +64,12 @@ Database Features
    * - Database Monitoring
      - *Unsupported*
 
+   * - Multi-database Support / Multiple Schemas
+     - Partially supported. Laravel uses a dot separator (``.``)
+       between SQL schema and table names, but MongoDB allows ``.``
+       characters within collection names, which might lead to
+       unexpected namespace parsing.
```

Comment:
We don't have implemented anything to avoid breaking changes. I would not tell it's partially supported.